POSIX/demo/demo.cpp

```diff
@@ -414,6 +414,83 @@ void testExpectedCapacity() {
     }
 }
 
+void testOnlyOneKey() {
+    string key = "name";
+    string s;
+    {
+        s = "";
+        auto mmkv = MMKV::mmkvWithID("testOneKey");
+
+        mmkv->getString(key, s);
+        printf("value: value = %s\n", s.c_str());
+
+        mmkv->set("world", key);
+        mmkv->getString(key, s);
+        printf("value: value = %s\n", s.c_str());
+
+        for (int i = 0; i < 10; i++) {
+            string value = "world_";
+            value += to_string(i);
+            mmkv->set(value, key);
+            mmkv->getString(key, s);
+            printf("value: value = %s\n", s.c_str());
+        }
+
+        // test file expanding
+        string bigValue(100000, '0');
+        mmkv->set(bigValue, key);
+        mmkv->getString(key, s);
+        printf("value: value = %s\n", s.c_str());
+
+        mmkv->set("OK", key);
+        mmkv->getString(key, s);
+        printf("value: value = %s\n", s.c_str());
+    }
+
+    {
+        s = "";
+        string cryptKey = "fastest";
+        auto mmkv = MMKV::mmkvWithID("testOneKeyCrypt", MMKV_SINGLE_PROCESS, &cryptKey);
+
+        mmkv->getString(key, s);
+        printf("value: value = %s\n", s.c_str());
+
+        mmkv->set("cryptworld", key);
+        mmkv->getString(key, s);
+        printf("value: value = %s\n", s.c_str());
+
```

Comment:
For an encryption test, you need to close it in the middle (for example, after a second write with the same key). Reopen it to see if the decryption succeeded.

Reply:
OK. Actually I tested the case you mentioned by rerun the tests. I will add it to code above.